### ISSUE TITLE
Combining multiple runs

### ISSUE THREAD

**Issue by johnwhitington** [lpw25/prof_alloc#7]
It would be great to be able to have an option to merge the results of two or more dumps (assuming they were from the same executable).

We could then answer questions like "What are the biggest causes of slow compile times in `ocamlopt`?" or, in my case, "What are the biggest bottlenecks in `cpdf` across all of the test files"?

**Comment by lpw25** [comment]
Yeah, that's probably a good idea. Might need a bit of thought for the best way to combine the results. I'm not sure just adding them together makes sense for some of the data.

In the meantime, if you have the energy, the basically undocumented `prof_spacetime print` feature can be combined with some sed/grep/awk fiddling to get these kinds of numbers.
